content/pages/constitution.md

```diff
@@ -0,0 +1,313 @@
+Author: Fatema Boxwala
+Title: Constitution
+hidden: true
+
+# Structure of Organization #
+
+The Women in Computer Science Undergraduate Subcomittee is a
+subcommittee of Women in Computer Science, a standing committee of the
+Cheriton School of Computer Science. For the remainder of this document,
+this standing committee will be referred to as the "WiCS Committee". All
+references to: "the undergraduate committee", "the committee", and
+"committee members" refer to the Undergraduate Subcommittee unless
+otherwise specified.
+
+## Purpose and Vision ##
+
+The undergraduate committee aims to create a safe environment for cis
+and trans women as well as non-binary individuals in computer science,
+and to advocate for the inclusion of these marginalized groups in the
+computer science community at large.
```

Comment:
How do people feel about changing the first sentence in Purpose and Vision to: "The Undergraduate Committee aims to _support the success of cis and trans women and non binary individuals in computer science, by fostering a safe environment for and advocating for the inclusion of these marginalized groups in the computer science community at large_."